docs/src/plasma/qtplasmac.adoc

```diff
@@ -3985,62 +4001,62 @@ There are some sample G-code files in the ~/linuxcnc/nc_files/examples/plasmac d
 [width="100%",cols="1,2",options="header"]
 |===
 |Description |Code
-|Begin <<qt_multi-tool,cut>> |M3 $0 S1
-|End <<qt_multi-tool,cut>> |M5 $0
-|Begin <<qt_scribe,scribe>> |M3 $1 S1
-|End <<qt_scribe,scribe>> |M5 $1
-|Begin <<qt_spotting,center spot>> |M3 $2 S1
-|End <<qt_spotting,center spot>> |M5 $2
-|End all the above |M5 $-1
-|Select a <<qt_material-handling,material>> |M190 P__n__ +
+|Begin <<qt_multi-tool,cut>> m|M3 $0 S1
+|End <<qt_multi-tool,cut>> m|M5 $0
+|Begin <<qt_scribe,scribe>> m|M3 $1 S1
+|End <<qt_scribe,scribe>> m|M5 $1
+|Begin <<qt_spotting,center spot>> m|M3 $2 S1
+|End <<qt_spotting,center spot>> m|M5 $2
+|End all the above m|M5 $-1
+|Select a <<qt_material-handling,material>> m|`M190 P`__n__ +
 _n_ denotes the material number.
-|Wait for <<qt_material-handling,material>> change confirmation |M66 P3 L3 Qn +
+|Wait for <<qt_material-handling,material>> change confirmation |`M66 P3 L3 Q`__n __+
 _n_ is delay time (in seconds). +
 This value may need to be increased for very large material files.
-|Set feed rate from <<qt_material-handling,material>> |F#<_hal[plasmac.cut-feed-rate]>
-|Enable <<qt_ignore-ok,Ignore Arc OK>> |M62 P1 (synchronized with motion) +
-M64 P1 (immediate)
-|Disable <<qt_ignore-ok,Ignore Arc OK>> |M63 P1 (synchronized with motion) +
-M65 P1 (immediate)
-|Disable <<qt_thc,THC>> |M62 P2 (synchronized with motion) +
-M64 P2 (immediate)
-|Enable <<qt_thc,THC>> |M63 P2 (synchronized with motion) +
-M65 P2 (immediate)
-|Disable <<qt_overcut,Torch>> |M62 P3 (synchronized with motion) +
-M64 P3 (immediate)
-|Enable <<qt_overcut,Torch>> |M63 P3 (synchronized with motion) +
-M65 P3 (immediate)
-|Set <<qt_velocity_thc,velocity>> to a percentage of feed rate |M67 E3 Qn (synchronized with motion) +
-M68 E3 Qn (immediate) +
+|Set feed rate from <<qt_material-handling,material>> |`F#<_hal[plasmac.cut-feed-rate]>
```

Comment:
I guess here is a `` ` `` missing or similar

Reply:
Have introduced an "m" and removed the "`".